client/components/domains/use-my-domain/utilities/get-domain-inbound-transfer-status-info.js

```diff
@@ -1,9 +1,9 @@
 import wpcom from 'calypso/lib/wp';
 
 export async function getDomainInboundTransferStatusInfo( domainName ) {
-	const inboundTransferStatusResult = await wpcom
-		.undocumented()
-		.getInboundTransferStatus( domainName );
+	const inboundTransferStatusResult = await wpcom.req.get( {
+		path: `/domains/${ encodeURIComponent( domainName ) }/inbound-transfer-status`,
+	} );
```

Comment:
We could simplify the `wpcom.req.get()` call by changing the object with `path` argument to just a string argument shorthand:

```suggestion
	const inboundTransferStatusResult = await wpcom.req.get(
		`/domains/${ encodeURIComponent( domainName ) }/inbound-transfer-status`
	);
```

Reply:
Thanks! I'll note that for future methods 😊